src/Microsoft.TemplateEngine.Core/Expressions/Cpp2/Cpp2StyleEvaluatorDefinition.cs

```diff
@@ -113,7 +113,7 @@ protected override ITokenTrie GetSymbols(IProcessorState processor)
                 trie.AddToken(processor.Encoding.GetBytes("<<"));
                 trie.AddToken(processor.Encoding.GetBytes(">>"));
 
-                //Maths
+                //Math operators
```

Comment:
missing space?

Reply:
It is very inconsistent. I updated in this file for now. I would think all the files should get fixed over time with edits (unless there is a tool to fix these)